samples/apps/bookinfo/rules/mixer-rule-standard-attributes.yaml

```diff
@@ -84,4 +106,16 @@ spec:
         valueType: STRING
       target.serviceAccount:
         valueType: STRING
+      destination.ip:
```

Comment:
Please add `destination.port` if you are at it.

Reply:
needs to be done in the API repo first.